contracts/script/parsers/EigenlayerContractsParser.sol

```diff
@@ -10,19 +10,21 @@ import {IStrategyManager, IStrategy} from "eigenlayer-contracts/src/contracts/in
 import {ISlasher} from "eigenlayer-contracts/src/contracts/interfaces/ISlasher.sol";
 import {StrategyBaseTVLLimits} from "eigenlayer-contracts/src/contracts/strategies/StrategyBaseTVLLimits.sol";
 import {IRewardsCoordinator} from "eigenlayer-contracts/src/contracts/interfaces/IRewardsCoordinator.sol";
+import {IAllocationManager} from "eigenlayer-contracts/src/contracts/interfaces/IAllocationManager.sol";
 
 import {ConfigsReadWriter} from "./ConfigsReadWriter.sol";
 import "forge-std/StdJson.sol";
+import {IAllocationManager} from "../../lib/eigenlayer-middleware/lib/eigenlayer-contracts/src/contracts/interfaces/IAllocationManager.sol";
 
-struct EigenlayerContracts {
+    struct EigenlayerContracts {
     ProxyAdmin eigenlayerProxyAdmin;
     PauserRegistry eigenlayerPauserReg;
     IStrategyManager strategyManager;
     IDelegationManager delegationManager;
-    ISlasher slasher;
     IAVSDirectory avsDirectory;
     IRewardsCoordinator rewardsCoordinator;
     StrategyBaseTVLLimits baseStrategyImplementation;
+    IAllocationManager allocationManager;
```

Comment:
nit: should we add permission controller as well?

Reply:
I want to first fix it for existing test. but yes we can add that.